src/cli/gentest/source_code_generator.py

```diff
@@ -17,13 +17,13 @@
 template_env = jinja2.Environment(loader=template_loader, keep_trailing_newline=True)
 
 # This filter maps python objects to string
-template_env.filters["stringify"] = lambda input: repr(input)
+template_env.filters["stringify"] = lambda value: repr(value)
 
 
 # generates a formatted pytest source code by writing provided data on a given template.
 def get_test_source(provider: Provider, template_path: str) -> str:
     """
-    Generates formatted pytest source code by rendering a template with provided data.
```

Comment:
An example of a docstring that is **not** using imperitive mood. Ruff asks us to change the latter.